### DISCUSSION TITLE
Public facing "Site wide" messages/notifications

### DISCUSSION THREAD

**Discussion by sofietoft** [GitbookIO/community#13] [Feature Requests]
### What's your feature/idea?

Being able to add a message (public-facing site) on Organization, Collection, and/or Space level.
It should be possible to add the following content to the message:
- Text
- Images
- Linkspublic-facing

### What's your desired outcome?

Use cases:

- We would like to be able to use our documentation platform as a way to advertise and share information about things happening in our community.
- Upcoming releases (new minor every 6 weeks, new major every 6 months)

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by jamescarppe** [comment]
Seconded. We use spaces within a collection for documentation on each version of our product we release (one space per version), and we would love to be able to put a banner at the top of each page in the non-main space to tell users "You are viewing documentation for an older version. To view the latest version documentation, click here" (with a link to the main space).

**Comment by dot-alert** [comment]
Thirded.
This is very much needed.

**Comment by sofietoft** [comment]
Is this something GitBook will consider at some point?
I can see that a "pa-integrations" label has been added - what exactly does that means for this feature request?
